### PR TITLE
Update aliases from nri -> @mswjs/interceptors

### PR DESCRIPTION
**Pull request by msutkowski**
Updates aliases and GitHub issues to be representative of the new package name

### CODE REVIEW

File: src/interceptors/ClientRequest/utils/normalizeHttpRequestParams.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ function resolveRequestOptions(
   /**
    * Clone the request options to lock their state
    * at the moment they are provided to `ClientRequest.
-   * @see https://github.com/mswjs/node-request-interceptor/issues/86
+   * @see https://github.com/@mswjs/interceptors/issues/86
```
Comment (kettanaito) [inline src/interceptors/ClientRequest/utils/normalizeHttpRequestParams.ts:29]: You don't have to include `@` in the organization name. This should be `mswjs/interceptors` instead.

**Review by kettanaito — CHANGES_REQUESTED** [review]
Thanks for addressing this, @msutkowski! 
If I may, just one point regarding the GitHub links.

File: src/interceptors/ClientRequest/utils/normalizeHttpRequestParams.ts (hunk 1 above)
Comment (msutkowski) [reply src/interceptors/ClientRequest/utils/normalizeHttpRequestParams.ts:29]: Bad find and replace on my part 🤦‍♂️. Will update.

**Comment by msutkowski** [comment]
> Thanks for addressing this, @msutkowski!
> If I may, just one point regarding the GitHub links.

All set, and confirmed the links work. Sorry about that 🤦 .

**Review by kettanaito — APPROVED** [review]
Gorgeous ✨ Thank you, Matt!

**Pull request merged**